src/skills-builder/skills-builder-modal/view-results/ViewResults.jsx

```diff
@@ -66,9 +69,35 @@ const ViewResults = () => {
       });
   }, [careerInterests, jobSearchIndex, productSearchIndex]);
 
+  const sendRecommendationShownEvent = useCallback((selectedJobName, isDefault) => {
+    sendTrackEvent('edx.skills_builder.recommendation.shown', {
+      app_name: 'skills_builder',
+      category: 'skills_builder',
+      page: 'skills_builder',
+      selected_recommendations: productRecommendations.find(rec => rec.name === selectedJobName),
+      is_default: isDefault,
+    });
+  }, [productRecommendations]);
+
   useEffect(() => {
-    setSelectedRecommendations(productRecommendations.find(rec => rec.name === selectedJobTitle));
-  }, [productRecommendations, selectedJobTitle]);
+    /*
+      This useEffect will fire when selectedJobTitle is changed
+      We initially setSelectedJobTitle when the recommendations are returned
+      We are sending an event with the is_default field set to true for the initial default selection
+    */
+    const newlySelectedRecommendations = productRecommendations.find(rec => rec.name === selectedJobTitle);
+    if (!selectedRecommendations && newlySelectedRecommendations) {
+      sendRecommendationShownEvent(selectedJobTitle, true);
+    }
+    setSelectedRecommendations(newlySelectedRecommendations);
+  }, [productRecommendations, selectedJobTitle, selectedRecommendations, sendRecommendationShownEvent]);
+
+  const handleJobTitleChange = (e) => {
+    const { value } = e.target;
+    setSelectedJobTitle(value);
+    // The is_default value will be set to false for any selections made by the user
+    sendRecommendationShownEvent(value, false);
+  };
```

Comment:
Newly added code for the `edx.skills_builder.recommendation.shown` event that will fire in two places:

1. Once for the default selection that is made by the app when the recommendations are first loaded. We default this selection to whichever set of recommendations is returned first in the list. For this event, the `is_default` field is set to `true`.
2. And then whenever a user makes a selection through the `onChange` handle for the `SelectableBox`(radio button) components. The `is_default` field is set to `false` whenever the user makes this selection.

The utility function that is re-used to fire both events is wrapped in a `useCallback` hook to avoid issues with the `useEffect` dependency array.